### PR TITLE
Add extra rules to notification report

### PR DESCRIPTION
**Pull request by ushkarev**

### CODE REVIEW

**Comment by ushkarev** [comment]
retest this please

File: mtp_api/apps/notification/rules.py [hunk 1]
```diff
@@ -114,6 +118,63 @@ def get_event_trigger(self, record):
...
+        return None
+
+    def triggered(self, record):
+        profile = self.get_event_trigger(record)
+        if not profile or profile == self.shared_profile:
+            return False
+
+        records_of_same_type = self.get_profile_records_of_same_type(profile, record)
+        field_to_count = self.kwargs['count']
+        count = records_of_same_type.values(field_to_count).distinct().count()
```
Comment (ibrechin) [inline mtp_api/apps/notification/rules.py]: If only this count was kept updated and stored somewhere...

Comment (ushkarev) [reply mtp_api/apps/notification/rules.py]: where do i click to report abuse

Comment (ibrechin) [reply mtp_api/apps/notification/rules.py]: "this guy keeps commenting on my PRs and being right about everything"

**Pull request merged**